react/week4/github-reasecher/src/context/Lounchpad.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import {  SearchContextProvider } from './createContext';
+import Header from './Header'
+import SearchInput from './SearchInput';
+import UserSearchResults from './UserSearchResults';
+
+const Lounchpad = () => {
+
+  const [users, setUsers] = useState([])
+  const [error, setError] = React.useState();
+  const [loading, setLoading] = useState(false)
+  const [textValue, setTextValue] = React.useState('');
+
+  React.useEffect(() => {
+    if (textValue !== "") {
+      setLoading(false);
```

Comment:
Isn't this supposed to be true?
```suggestion
      setLoading(true);
```